src/routes/docs/advanced/security/abuse-protection/+page.markdoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: article
+title: Abuse protection
+description: Learn how Appwrite protects your apps from abuse through rate limiting and cross-site scripting protection.
+---
+Appwrite comes packaged with tools to protect against various forms of abuse, like brute force attacks, data scraping, cross-site request forgery, and many
```

Comment:
Not sure our abuse protection is helpful against [CSRF)](https://owasp.org/www-community/attacks/csrf). What is helpful for that is our CORS settings and the fact each domain need to list allowed domains for enabling API communications. Thinking about it, it makes sense to have a section about CORS at Appwrite.